cmd/server/main.go

```diff
@@ -568,8 +568,11 @@ func (p *Proxy) runMTLSMasterServer(ctx context.Context, o *ProxyRunOptions, s *
 	addr := fmt.Sprintf(":%d", o.serverPort)
 
 	if o.mode == "grpc" {
-		serverOption := grpc.Creds(credentials.NewTLS(tlsConfig))
-		grpcServer := grpc.NewServer(serverOption)
+		serverOptions := []grpc.ServerOption{
+			grpc.Creds(credentials.NewTLS(tlsConfig)),
+			grpc.KeepaliveParams(keepalive.ServerParameters{Time: o.keepaliveTime}),
```

Comment:
I agree we should default to the same timeout for both tunnels but so we want to force the same timeout for both tunnels?

Reply:
Are you suggesting we add an additional param for this one?

Reply:
That or make an argument why we don't need it. :smiley_cat: